stale_repos.py

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python
 """ Find stale repositories in a GitHub organization. """
+from __future__ import annotations
```

Comment:
why do we need this?

Reply:
Hello @jmeridth,
I've tried this because of this error on the pipeline:
https://github.com/github/stale-repos/actions/runs/11576316507/job/32229613440#step:5:38
It seems that it's not the best approach since it still fails. I haven't looked into the other repositories to check what might be the difference because the code between them is the same

Reply:
I think it may be time to drop support for python v3.9 like we have for our other actions.